lib/content/getting-started.md

```diff
@@ -782,7 +776,7 @@ Hey, this is a pretty good start for Paldo—good work!  As you can see, there's
 - [the pal boilerplate](https://hapipal.com/docs/boilerplate) - this is the baseline setup for pal projects, including a nice setup deployment, testing, linting, and pluginization of your application.  It also offers a handful of "flavors", which helped us more easily integrate Swagger documentation and a SQL-backed model layer.
 - [hpal](https://hapipal.com/docs/hpal) - this is the command line tool we used to start a new project, create routes in `routes/`, and models in `models/`.  It does much more too—you can also search documentation with it from the command line, for example: `hpal docs:schwifty request.models`.
 - [haute-couture](https://hapipal.com/docs/haute-couture) - this is used by the pal boilerplate to enforce the directory structure for your hapi plugin (everything in `lib/`).
-- [joi](https://github.com/hapijs/joi/blob/master/API.md) - this is the validation library of choice for hapi projects, since it integrates nicely into hapi itself.
+- [joi](https://github.com/sideway) - this is the validation library of choice for hapi projects, since it integrates nicely into hapi itself.
```

Comment:
```suggestion
- [joi](https://github.com/sideway/joi) - this is the validation library of choice for hapi projects, since it integrates nicely into hapi itself.
```